### PR TITLE
Feature/sdx 1485 remove jquery

### PR DESCRIPTION
**Pull request by muthan**

### CODE REVIEW

File: src/framework/organizationSwitcher/OrganizationSwitcher.jsx [hunk 1]
```diff
@@ -14,11 +13,13 @@ export {
 class OrganizationSwitcher extends Component {
 
   componentWillMount() {
-    $('body').addClass('is-organization-switcher-open');
+    document.querySelector('body')
+    .classList.add('is-organization-switcher-open');
```
Comment (mariacasancho) [inline src/framework/organizationSwitcher/OrganizationSwitcher.jsx]: indentation

File: src/framework/organizationSwitcher/OrganizationSwitcher.jsx [hunk 2]
```diff
@@ -14,11 +13,13 @@ export {
...
   }
 
   componentWillUnmount() {
-    $('body').removeClass('is-organization-switcher-open');
+    document.querySelector('body')
+    .classList.remove('is-organization-switcher-open');
```
Comment (mariacasancho) [inline src/framework/organizationSwitcher/OrganizationSwitcher.jsx]: indentation

**Comment by arijitdasgupta** [comment]
Let's change jQuery dependency to dev dependency if we want to keep the examples jQuery dependent.

Or,

We change the Example files as well.

`src/guide/views/grid/GridExample.jsx`
`src/guide/views/gridView/GridExample.jsx`
`src/guide/components/sourceCodeViewer/SourceCodeViewer.jsx`
`src/guide/views/AppView.jsx`

Still has jQuery as a hard dependency.

**Comment by muthan** [comment]
:+1: PRA

**Comment by arijitdasgupta** [comment]
Made minor changes, looks good. 👍

**Pull request merged**